src/tracks/ui/TextEditDelegate.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+class wxString;
+class wxPoint;
+
+class AudacityProject;
+
+//Used as a delegate for TextEditHelper
+class TextEditDelegate
```

Comment:
Perhaps "TextEditPolicy".  This looks like the "Policy" pattern, also called "Strategy," where an object supplies a set of callbacks to a reusable framework.

Reply:
No, it's not a policy. It could be a `TextEditListener` or replaced with wxEvent dispatch